src/commonMain/kotlin/fr/acinq/lightning/io/Peer.kt

```diff
@@ -566,6 +568,31 @@ class Peer(
                     // no additional inputs or outputs, the splice is only meant to bump fees
                     spliceIn = null,
                     spliceOut = null,
+                    requestRemoteFunding = null,
+                    feerate = feerate
+                )
+                send(WrappedChannelCommand(channel.channelId, spliceCommand))
+                spliceCommand.replyTo.await()
+            }
+    }
+
+    suspend fun estimateFeeForInboundLiquidity(amount: Satoshi, feerate: FeeratePerKw): LiquidityAds.LeaseFees {
```

Comment:
This should take a `targetFeerate` and return the corrected feerate (taking into account unconfirmed parents), like we do for splice-out and cpfp. (also the ordering of methods is not consistent)

```suggestion
    suspend fun estimateFeeForInboundLiquidity(amount: Satoshi, targetFeerate: FeeratePerKw): Pair<FeeratePerKw, LiquidityAds.LeaseFees> {
```

Reply:
Good idea, done in https://github.com/ACINQ/lightning-kmp/pull/561/commits/cb0c0f80900c95f03f4be9c335decc046229958b